### PR TITLE
latest changes

### PR DESCRIPTION
**Pull request by LakshmiKumar23**
Includes both Hansel and my changes

### CODE REVIEW

File: amd_openvx/openvx/ago/ago_platform.h [hunk 1]
```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
 
 // OpenCL: enabled unless disabled explicitly by setting ENABLE_OPENCL=0
 #ifndef ENABLE_OPENCL
-#define ENABLE_OPENCL  1
```
Comment (kiritigowda) [inline amd_openvx/openvx/ago/ago_platform.h]: This needs be turned on by default. Only use -D option in CMake to turn off while building.

File: amd_openvx/openvx/include/VX/vx_compatibility.h [hunk 2]
```diff
@@ -46,8 +46,10 @@
 #define VX_CONTEXT_ATTRIBUTE_EXTENSIONS                         VX_CONTEXT_EXTENSIONS
 #define VX_CONTEXT_ATTRIBUTE_CONVOLUTION_MAXIMUM_DIMENSION      VX_CONTEXT_CONVOLUTION_MAX_DIMENSION
 #define VX_CONTEXT_ATTRIBUTE_OPTICAL_FLOW_WINDOW_MAXIMUM_DIMENSION      VX_CONTEXT_OPTICAL_FLOW_MAX_WINDOW_DIMENSION
-#define VX_CONTEXT_ATTRIBUTE_IMMEDIATE_BORDER_MODE                      VX_CONTEXT_IMMEDIATE_BORDER
```
Comment (kiritigowda) [inline amd_openvx/openvx/include/VX/vx_compatibility.h]: We should not edit the header files, they come from Khronos.

https://www.khronos.org/registry/OpenVX/

Comment (LakshmiKumar23) [reply amd_openvx/openvx/include/VX/vx_compatibility.h]: @kiritigowda The new version asks to query context for 3-4 new things that were not in the previous versions. They are not included in the header file. How do I go about this?

Comment (kiritigowda) [reply amd_openvx/openvx/include/VX/vx_compatibility.h]: We can add this to vx_ext_amd.h

**Review by kiritigowda — APPROVED** [review]

**Pull request merged**